python/orca/test/bigdl/orca/tfpark/test_tf_dataset.py

```diff
@@ -325,7 +325,7 @@ def test_tfdataset_with_tf_data_dataset_which_contains_bool(self):
         dataset = tf.data.Dataset.from_tensor_slices((np.random.randn(102, 28, 28, 1),
                                                       np.random.randint(0, 10, size=(102,)),
                                                       np.ones(shape=(102, 28, 28, 1),
-                                                              dtype=np.bool)))
+                                                              dtype=bool)))
```

Comment:
`tfpark` is only tested in python 3.7. Here the code change is just for consistency